src/app/visual/colorpicker/colorpicker-visual.component.html

```diff
@@ -2,26 +2,39 @@
   (themeSettingsChange)="themeSettingsChange($event)"
 ></sky-docs-demo-control-panel-theme>
 
-<div
-  id="screenshot-colorpicker"
+<button
+  class="sky-btn sky-btn-default"
+  id="screenshot-action-button-center-align"
+  type="button"
+  (click)="onToggleAbleColorpicker()"
 >
-  <sky-colorpicker
-    labelledBy="FOOBAR"
-    #colorPickerExample1
+  Toggle Colorpicker #1's Ability
+</button>
+
+<form
+  [formGroup]="myForm"
```

Comment:
Now that we're adding more types of colorpickers to the visual test page, would you mind adding some headers to provide some clarity? We could do something like selection box with `<h3>`'s: https://github.com/blackbaud/skyux-forms/blob/master/src/app/visual/selection-box/selection-box-visual.component.html#L9

Reply:
done in last commit